runbooks/koji/upgrade.md

```diff
@@ -0,0 +1,83 @@
+Koji upgrade
+==============
+
+Issue
+-----
+* You want to upgrade [Koji](http://koji.katello.org) to a newer version
+
+Resolution
+----------
+* Shut down koji services on `koji-fedora28-builder` and `kojibuilder1`.
+  * `koji-fedora28-builder`
+    ```
+    # systemctl stop kojid
+    ```
+  * `kojibuilder1`
+	```
+    # systemctl stop kojid
```

Comment:
Odd indenting. Is there also a reason to stop these sequentially instead of `systemctl stop kojid kojira httpd postgresql`

Reply:
mostly just expressing the appropriate order via commands, as stopping postgres or http before kojira or kojid would cause them to fail